src/app/app.module.ts

```diff
@@ -30,45 +30,47 @@ export function initializeApp(appInitService: AppInitService) {
 }
 
 @NgModule({
-    declarations: [
-        AppComponent,
-        ModifyComponent,
-        ReadComponent,
-        ActionPlaygroundComponent,
-        SearchPlaygroundComponent,
-        SearchResultsComponent,
-        AdvancedSearchPlaygroundComponent
-    ],
-    imports: [
-        BrowserModule,
-        BrowserAnimationsModule,
-        AppRoutingModule,
-        DspCoreModule,
-        DspViewerModule,
-        DspActionModule,
-        DspSearchModule,
-        MatJDNConvertibleCalendarDateAdapterModule,
-        MatButtonModule,
-        MatListModule
-    ],
-    providers: [
-        AppInitService,
-        {
-            provide: APP_INITIALIZER,
-            useFactory: initializeApp,
-            deps: [AppInitService],
-            multi: true
-        },
-        {
-            provide: DspApiConfigToken,
-            useFactory: () => AppInitService.dspApiConfig
-        },
-        {
-            provide: DspApiConnectionToken,
-            useFactory: () => AppInitService.dspApiConnection
-        }
-    ],
-    bootstrap: [AppComponent]
+  declarations: [
+    AppComponent,
+    ModifyComponent,
+    ReadComponent,
+    ActionPlaygroundComponent,
+    SearchPlaygroundComponent,
+    SearchResultsComponent,
+    AdvancedSearchPlaygroundComponent
+  ],
+  imports: [
+    BrowserModule,
+    BrowserAnimationsModule,
+    AppRoutingModule,
+    DspCoreModule,
+    DspViewerModule,
+    DspActionModule,
+    DspSearchModule,
+    MatJDNConvertibleCalendarDateAdapterModule,
+    MatButtonModule,
+    MatListModule
+  ],
+  providers: [
+    {
+      provide: APP_INITIALIZER,
+      useFactory: initializeApp,
+      deps: [AppInitService],
+      multi: true
+    },
+    {
+      provide: DspApiConfigToken,
+      useFactory: (appInitService: AppInitService) => appInitService.dspApiConfig,
+      deps: [AppInitService]
+    },
+    {
+      provide: DspApiConnectionToken,
+      useFactory: (appInitService: AppInitService) => new KnoraApiConnection(appInitService.dspApiConfig),
+      deps: [AppInitService]
+    }
+  ],
+  bootstrap: [AppComponent]
+
```

Comment:
Indent should be 4 spaces in typescript files.

Reply:
fixed in https://github.com/dasch-swiss/knora-ui-ng-lib/pull/118/commits/262c9545e62ba2b3d947f5b3b63b96bb8b5b3daf